packages/eddsa-proof/tests/index.test.ts

```diff
@@ -14,6 +14,7 @@ describe("EddsaProof", () => {
 
     beforeAll(async () => {
         curve = await buildBn128()
+        fullProof = await generate(privateKey, scope)
```

Comment:
if accessing an outer scope variable in both tests, we need to make sure it is setup before both of them. As jest runs tests concurrently, I have experienced failing tests because the 2nd test would run before the 1st and fullProof wasn't defined yet.

Reply:
Umh I guess it would be better to have independent tests in general. Generating a proof takes some time tho, so in this case it could make sense. Isn't it possible to run those tests sync?